app/controllers/organisation_relationships_controller.rb

```diff
@@ -14,7 +14,7 @@ class OrganisationRelationshipsController < ApplicationController
   ]
 
   def stock_owners
-    stock_owners = organisation.stock_owners.filter_by_active
+    stock_owners = organisation.stock_owners.filter_by_active.order("LOWER(name)")
```

Comment:
Can there be some tests for this sorting?

Reply:
Sure will add some